packages/vc-verification/test/issuer-verification-test.ts

```diff
@@ -627,6 +627,78 @@ function testSuite() {
       ).to.be.true;
     });
 
+    it('invalid RoleEIP191Jwt should not break verification,', async () => {
+      const adminJWT = new JWT(adminKeys);
+      const claimAdmin = {
+        claimData: { fields: {}, claimTypeVersion: 1 },
+        iss: adminDid,
+        credentialStatus: {
+          id: 'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+          type: 'StatusList2021Entry',
+          statusPurpose: 'revocation',
+          statusListIndex: '0',
+          statusListCredential:
+            'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+        },
+        signer: adminDid,
+      };
+      const token = await adminJWT.sign(claimAdmin);
+      const ipfsCIDAdmin = await didStore.save(token);
+      const serviceIdAdmin = adminRole;
+      const updateDataAdmin: IUpdateData = {
+        type: DIDAttribute.ServicePoint,
+        value: {
+          id: `${adminDid}#service-${serviceIdAdmin}`,
+          type: 'ClaimStore',
+          serviceEndpoint: ipfsCIDAdmin,
+        },
+      };
+      await adminOperator.update(
+        adminDid,
+        DIDAttribute.ServicePoint,
+        updateDataAdmin,
+        validity
+      );
+
+      const claimManager = {
```

Comment:
Seems like an odd name for the variable. What do you think of this instead?
```suggestion
      const managerRoleClaim = {
```